src/components/profile/sharebutton.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { SimpleShareButtons } from "react-simple-share";
+
+export const sharebutton = () => (
+  <span className="sharebuttons">
+    <SimpleShareButtons
+      color="#FFFFFF"
+      size="30px"
+      whitelist={["Facebook", "Twitter"]}
```

Comment:
Can we add instagram or whatsapp if there is too?

Reply:
seems like there's nothing for instagram share.
But this has for whatsapp...thinking if it's necessary still